css/style.css

```diff
@@ -0,0 +1,2890 @@
+    background: url(../img/model-s-bottom.png) no-repeat;
+    background-position: 470px 142px;
+    background-size: 370px 100px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-position: 430px 150px;
+    background-size: 450px auto;
+  }
+
+}
+
+@media (max-width: 880px) {
+  .container {
+    max-width: 880px;
+  }
+
+  .top__wrapper-charset {
+    margin-right: 0;
+  }
+
+  .charset__speed {
+    margin-left: 100px;
+  }
+
+  .top__wrapper-item {
+    margin-right: 110px;
+  }
+
+  .button__outline {
+    padding-left: 0;
+    padding-right: 20px;
+  }
+
+  .button__red {
+    padding-left: 25px;
+    padding-right: 25px;
+    margin-right: 0;
+  }
+
+  .middle {
+    max-width: 880px;
+    margin-top: 30px;
+  }
+
+  .middle__inner {
+    align-content: space-between;
+    background: url(../img/tesla-fon-ipad-2.png) no-repeat;
+    background-size: 880px auto;
+  }
+
+  .middle__fon {
+    display: flex;
+    display: inline-block;
+  }
+
+  .middle__title {
+    margin-top: 11px;
+    margin-left: 434px;
+  }
+
+  .middle__info {
+    max-width: 880px;
+    height: 490px;
+    margin-top: 405px;
+  }
+
+  .middle__info-row {
+    display: flex;
+    flex-direction: row;
+  }
+
+  .middle__content {
+    margin-left: 0;
+  }
+
+  .middle__info-title {
+    margin-top: 57px;
+    margin-left: 32px;
+  }
+
+  .middle__info-text {
+    width: 320px;
+    margin-right: 0;
+    margin-top: 22px;
+    margin-left: 32px;
+  }
+
+  .middle__row {
+    margin-right: 70px;
+    margin-top: 33px;
+    margin-left: 50px;
+  }
+
+  .middle__screen {
+    padding-bottom: 0;
+    padding-top: 8px;
+  }
+
+  .middle__text {
+    padding-bottom: 25px;
+  }
+
+  .middle__btn {
+    margin-left: 152px;
+    margin-top: 17px;
+  }
+
+  .button__btn {
+    display: flex;
+    align-items: center;
+    margin-left: 150px;
+    margin-right: 0;
+    padding-left: 0;
+    margin-top: 15px;
+  }
+
+  .button__btn-img {
+    width: 32px;
+    height: 32px;
+  }
+
+  .button__btn-word {
+    width: 150px;
+    margin-left: 15px;
+    margin-right: 23px;
+    font-weight: 600;
+    font-size: 16px;
+    line-height: 20px;
+  }
+
+  .button__btn-text {
+    font-size: 16px;
+    line-height: 20px;
+    margin-left: 20px;
+    padding-left: 63px;
+    padding-right: 63px;
+    padding-top: 15px;
+    padding-bottom: 15px;
+  }
+
+  .bottom {
+    background: url(../img/model-s-fon-ipad.png) no-repeat;
+    background-position: 95px 68px;
+  }
+
+  .bottom__inner {
+    background: url(../img/car-ipad.png) no-repeat;
+    background-position: 0 75px;
+  }
+
+  .bottom__inner-img {
+    padding-top: 75px;
+  }
+
+  .bottom__info {
+    height: 490px;
+    width: 880px;
+    margin-top: 538px;
+  }
+
+  .bottom__content-stars {
+    display: flex;
+    height: 365px;
+  }
+
+  .bottom__description {
+    margin-right: 0;
+    padding-left: 35px;
+    padding-top: 62px;
+    max-width: 362px;
+    height: 355px;
+  }
+    
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/vovafinal/pullRequest?prid=5998857)